### DISCUSSION TITLE
Encouraging “Uses page” and “Brag documents”

### DISCUSSION THREAD

**Discussion by dtinth** [wonderfulsoftware/webring#135] [Ideas]
Currently we have the RSS feed section to encourage blogging. It might be useful to add 2 more sections:

- The uses page
- The brag document

See sections below to learn what they mean…

# 📣 To webring members

If you have a “uses” or “brag” pages, please comment in this discussion with the URL to your pages. I might create a new section on the webring home page to show these pages more prominently!

- The URL should reside on your domain. You can redirect to other domains like GitHub, Medium, Google Docs. 
- If you build your own page, please include the “last updated” time on your webpage. Recommended markup: `<time datetime="2011-11-18" class="dt-updated">2011-11-18</time>`

# 🛠️ What is the “uses” page?

The uses page is for sharing your setup. People can learn about what tools you use.

⭐ Recommended reading:
- [Let's share your setup in /uses page](https://heyfirst.co/blog/lets-share-your-setup-in-uses-page)

Examples from the webring:
- https://garden.narze.live/Uses/
- https://heyfirst.co/uses

# 🗣️ What is the “brag document?”

The brag document is for sharing your accomplishments in an informal way.

⭐ Recommended watching:
- [Doing Great Work at Your Job isn't Enough. You'll Need to Brag Yourself. by Kan Ouivirach](https://www.youtube.com/watch?v=xv3XnW_eDJI)

Recommended readings:
- [ประโยชน์ของการเขียน Brag Document by Kan Ouivirach](https://medium.com/odds-team/%E0%B8%9B%E0%B8%A3%E0%B8%B0%E0%B9%82%E0%B8%A2%E0%B8%8A%E0%B8%99%E0%B9%8C%E0%B8%82%E0%B8%AD%E0%B8%87%E0%B8%81%E0%B8%B2%E0%B8%A3%E0%B9%80%E0%B8%82%E0%B8%B5%E0%B8%A2%E0%B8%99-brag-document-bde6b0bc57bb)
- [Get your work recognized: write a brag document](https://jvns.ca/blog/brag-documents/)
- [Brag now, remember later: Document your accomplishments by Monica Powell](https://github.com/readme/guides/document-success)

Examples from webring members:
- https://notes.dt.in.th/Brag
- https://github.com/mikkipastel/brag-document
- https://garden.narze.live/Brag

Examples:
- https://github.com/zkan/brag-documents

**Comment by narze** [comment]
My work-in-progress Brag doc: https://garden.narze.live/Brag
(I just realised my url is case sensitive 🥶)
